python/src/aiconfig/editor/client/src/utils/promptUtils.ts

```diff
@@ -81,6 +82,8 @@ export const PROMPT_SCHEMAS: Record<string, PromptSchema> = {
   "dall-e-2": DalleImageGenerationParserPromptSchema,
   "dall-e-3": DalleImageGenerationParserPromptSchema,
 
+  "ClaudeBedrockModelParser": ClaudeBedrockPromptSchema,
```

Comment:
So just to be clear, the "model_name" is "Claude" when we register it in the core model parsers?
Do we also need to do `"Claude": ClaudeBedrockPromptSchema`?

Reply:
yes we would need to register it as well

Reply:
discussed offline, we'll get rid of "Claude" for now